app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestAgeForAnimal:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17])
+        ]
+    )
+    def test_get_age_correctly(self, cat_age: int, dog_age: int, expected: list) -> None:
+        """Test if the function correctly converts cat and dog ages to human years."""
+        assert get_human_age(cat_age, dog_age) == expected
+
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            (-1, 5, [0, 0]),
+            (5, -1, [0, 0]),
+            (0, 5, [0, 0]),
+            (5, 0, [0, 0]),
+            (1000, 2000, [246, 397])
```

Comment:
The expected result for extremely high ages (1000, 2000) is `[246, 397]`. Ensure that the `get_human_age` function is designed to handle such large inputs and that these expected values are correct according to the function's logic.